examples/scipy/steps/importer/importer_step.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import os
+import re
+
+import numpy as np
+import pandas as pd
+
+from zenml.steps import Output, step
+
+TRAIN_PATH = os.path.join(os.path.dirname(__file__), "../../data", "train.csv")
```

Comment:
This joining might not work on windows due to the explicit `/` in the path. Can we use `pathlib` or `os.path.join`/`os.path.dirname` to get the correct path on all operating systems?